build.gradle.kts

```diff
@@ -118,3 +131,18 @@ publishing {
 		}
 	}
 }
+
+tasks.withType<Jar> {
+	enabled = true
+}
+
+tasks.withType<BootJar> {
+	enabled = true
+}
+
```

Comment:
add tasks to produce also source jar and javadoc jar:
https://stackoverflow.com/questions/11474729/how-to-build-sources-jar-with-gradle